reports-scheduler/.codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
```

Comment:
Do we need the copyright notice in `yml` files?

Reply:
added